ctms/models.py

```diff
@@ -240,18 +222,8 @@ class ApiClient(Base):
     enabled = Column(Boolean, default=True)
     hashed_secret = Column(String, nullable=False)
 
-    create_timestamp = Column(
-        TIMESTAMP(timezone=True), nullable=False, server_default=now()
-    )
-    update_timestamp = Column(
-        TIMESTAMP(timezone=True),
-        nullable=False,
-        server_default=now(),
-        server_onupdate=now(),
-    )
-
 
-class MozillaFoundationContact(Base):
+class MozillaFoundationContact(Base, TimestampMixin):
```

Comment:
This model didn't have any but I guess it's better for consistency

Reply:
Yeah that's what I thought as well. Why not have it.